### DISCUSSION TITLE
I have already written "sizes" attribute in Image, but it shows still warning.

### DISCUSSION THREAD

**Discussion by LuYujin9** [neuefische/questions#551] [Web] [answered]
the warning is 
"Image with src "/images/Carolaschlösschen1.jpg" has "fill" but is missing "sizes" prop. Please add it to improve page performance."

my code is
` <StyledPicture>
          <StyledImage
            alt="a photo of the restaurant"
            src={restaurant.photos[0]}
            fill
            sizes="100vw"
            priority
          />
        </StyledPicture>`

**Comment by PReichetanz** [comment]
Hi Jin,
please provide us with a link to the code so that we can try it locally.

**Comment by marcbendixen** [comment, marked as answer]
Hi @LuYujin9,

in the documentation you can find an example for `sizes`: https://nextjs.org/docs/api-reference/next/image#sizes
It's defined with breakpoints, maybe that's the missing part you need to add.

Example from Next docs:
```jsx
sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
```

**Reply by LuYujin9** [reply]
Thanks.
 It works, when I write like this :
`sizes="(max-width: 768px) 100vw"`
I tried it before, but it doesn't work. Maybe there was  a mistake in some other place.
but it's super that my code runs good now.
